src/types.ts

```diff
@@ -8,6 +9,29 @@ export interface TagInfo {
   description: string;
 }
 
+export interface CurveLpToken extends TokenInfo {
+  tags: [TokenTag.CURVE, ...string[]];
```

Comment:
minor: Rename `CurveLpTokenInfo` and remove this `tags` since specifying order makes it hard for token infos to extend each other. We should just need to specify the `extensions` here like we do for the other token infos.